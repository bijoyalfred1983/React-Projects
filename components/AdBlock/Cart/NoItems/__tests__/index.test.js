import React from 'react';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import { NoItemsComponent as NoItems } from '../index';

jest.mock('react-router-dom', () => ({
  Link:() => null
}));

describe('<NoItems /> container', () => {
  test('renders without breaking', () => {
  	const staticLabelData = {'test':'test'};
    const wrapper = mount(<NoItems staticLabelData={staticLabelData}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
