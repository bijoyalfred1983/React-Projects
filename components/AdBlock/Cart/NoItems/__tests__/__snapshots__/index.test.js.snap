// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NoItems /> container renders without breaking 1`] = `
<NoItemsComponent>
  <div
    className="container"
  >
    <img
      alt="empty"
      height="120"
      src="/images/cart_empty_icon.svg"
      width="120"
    />
    <div
      className="cartIsEmpty"
    >
      Looks like your cart is empty
    </div>
    <div
      className="searchProducts"
    >
      Search for Products Above
    </div>
    <Link
      to="/"
    />
  </div>
</NoItemsComponent>
`;
